src/workflow/backup.c

```diff
@@ -81,11 +80,8 @@ bool workflow_backup_create(const CreateBackupRequest* request)
         return false;
     }
 
-    // Wait for sd card.
-    const InsertRemoveSDCardRequest sd = {
-        .action = InsertRemoveSDCardRequest_SDCardAction_INSERT_CARD,
-    };
-    sdcard_handle(&sd);
+    // Wait for sd card
+    rust_workflow_sdcard_blocking(true);
```

Comment:
Would be nice readability-wise to use an enum like before, `SDCARD_ACTION_INSERT` and `SDCARD_ACTION_REMOVE`.